.typos.toml

```diff
@@ -1,11 +1,15 @@
 [files]
 extend-exclude = [
+    # It does not like "intoto_*.proto"
+    "external/googleapis/protolists/grafeas.list",
```

Comment:
In a future PR we should we exclude `external/googleapis/protolists/*`?

Reply:
Sure.